pkg/solana/client/multinode/transaction_sender.go

```diff
@@ -28,31 +28,32 @@ var (
 // (e.g. Successful, Fatal, Retryable, etc.)
 type TxErrorClassifier[TX any] func(tx TX, err error) SendTxReturnCode
 
-type sendTxResult struct {
+type sendTxResult[RESULT any] struct {
 	Err        error
 	ResultCode SendTxReturnCode
+	Result     *RESULT
```

Comment:
Typically generics are single letters, unless they need to be longer to distinguish from one another. In this case it is easy to see the only usage is the field named `Result`:
```suggestion
type sendTxResult[R any] struct {
	Err        error
	ResultCode SendTxReturnCode
	Result     *R
```

Reply:
Replied to your other comment, will resolve this one.